pyrdfstore/store.py

```diff
@@ -156,7 +156,10 @@ def select(self, sparql: str, named_graph: Optional[str] = None) -> Result:
         else:
             select_graph = Graph(store=self.sparql_store)
         result: Result = select_graph.query(sparql)
-        log.debug(f"from SPARQLStore :: {type(result)=} -> {result=}")
+        assert isinstance(result, Result), (
+            "Failed getting proper result for:" f"{sparql=}, got {result=}"
```

Comment:
- What does the `=` do in the curly braces?
- Why not make this one f-string instead of implicitly concatenating two strings?

Reply:
1/
newly discovered `=` in f strings automatically provides the variable-name as context into the generated output
so: `f"{num=}"` is the equivalent of `f"num={num}"`  -- nifty he?
(and variants by adding spaces and stuff around!)

2/ 
splitting the string into two is guidance to lint-fix if not it turns the line-breaking around into (the less readable variant imho)
```
assert isinstance(
    result, Result
), f"Failed getting proper result for: {sparql=}, got {result=}"
```